### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by WSoS** [AtomLinter/linter-clang#74]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (/usr/share/atom/resources/app.asar/src/package.js:806:9)
Package.hasActivationCommands (/usr/share/atom/resources/app.asar/src/package.js:731:20)
<unknown> (/usr/share/atom/resources/app.asar/src/package.js:185:24)
Package.measure (/usr/share/atom/resources/app.asar/src/package.js:163:15)
Package.load (/usr/share/atom/resources/app.asar/src/package.js:177:12)
PackageManager.loadPackage (/usr/share/atom/resources/app.asar/src/package-manager.js:355:14)
```

**Comment by k2b6s9j** [comment]
Dupe of #65

**Comment by kevinsawicki** [comment]
Fixed by https://github.com/AtomLinter/linter-clang/pull/61 and published in version `2.25.2`
